dali/CMakeLists.txt

```diff
@@ -50,9 +50,9 @@ endif()
 cuda_add_library(${dali_lib} SHARED ${DALI_SRCS} ${DALI_PROTO_OBJ})
 
 # Define symbol version script for libdali.so
-set(exports_map_path "${PROJECT_BINARY_DIR}/stage")
```

Comment:
Idea was not to pollute main PROJECT_BINARY_DIR, but store this intermediate files in separate dir.